behave_django/management/commands/behave.py

```diff
@@ -118,12 +128,19 @@ def add_arguments(self, parser):
     def handle(self, *args, **options):
 
         # Check the flags
-        if options['use_existing_database'] and options['simple']:
+        if options['use_existing_database'] and \
+                (options['simple'] or options['test_runner']):
             self.stderr.write(self.style.WARNING(
-                '--simple flag has no effect'
+                '--simple and --test-runner flags has no effect'
                 ' together with --use-existing-database'
             ))
 
+        elif options['simple'] and options['test_runner']:
+            self.stderr.write(self.style.WARNING(
+                '--test-runner flag has no effect'
+                ' together with --simple'
+            ))
+
```

Comment:
You need to adapt and extend the test for your changes.

Reply:
Sounds reasonable, but do you have a tests for manage.py command?

Reply:
Sure, take a look at [tests/unit/test_cli.py](https://github.com/behave/behave-django/blob/master/tests/unit/test_cli.py).

Reply:
yeah, sorry, my bad, I completely missed it.